### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by uncleZ333** [OnionUI/Onion#1187] [Q&A]
can i just buy a sd card off someone lol this is to complicated

**Comment by Aemiii91** [comment]
It's really not. It's just formatting a drive and copying some files. Did you try following this: https://onionui.github.io/docs/installation

**Reply by uncleZ333** [reply]
i follow youtube guides but they make you download roms yourself, i cant figure out how to get the ps1 games to add

**Reply by Aemiii91** [reply]
Make sure your roms are `chd` files, copy to `Roms/PS`, refresh roms on device, voila 👌

**Reply by uncleZ333** [reply]
you are a god thank you so much idk how you explained it better than videos but i got it to work

**Reply by Aemiii91** [reply]
Nice! 🎉

**Reply by uncleZ333** [reply]
should i add anything to bios though?

**Reply by Aemiii91** [reply]
Yes, that's right. The required files are detailed here: https://onionui.github.io/docs/emulators/psx

**Comment by uncleZ333** [comment]
like loading a game pops up add ps to bios for better compatability
